config/locales/en.yml

```diff
@@ -78,6 +87,22 @@ en:
         subtitle: Teaser
         summary: iTunes Summary
         title: Title
+      feed:
+        title: Feed Label
```

Comment:
I feel like this is referred to as "Title" in another piece of help text?  Ah well, can cleanup later.